gwpy/io/datafind.py

```diff
@@ -132,7 +136,10 @@ def _read_ffl_cache(self, site, tag):
             def _update_metadata(entry):
                 return type(entry)(site, tag, entry.segment, entry.path)
             with open(path, 'r') as fobj:
-                cache = list(map(_update_metadata, _iter_cache(fobj)))
+                cache = list(map(
+                    _update_metadata,
+                    _iter_cache(fobj, gpstype=float),
```

Comment:
Just a curiosity question: can `gpstype` be any valid python type (or `LIGOTimeGPS) or must it be only one of `LIGOTimeGPS` or `int`?

Reply:
Officially it can be anything, but in practice you would only want `LIGOTimeGPS`, `float`, or `int`.